minhphan156/Abode
# SJSU-1067_2 check booking for not logged in user

What I did:
- added new page BookingNotLoggedIn
- allow user to access private history route when authenticated with bookingID and lastName
- changed backend bookin.js to work with current historyOverView

How to test:
- use **5cc80f24e85a88435461775a** as booking id and **Stark** as last name to check booking
